config/crd/bases/openstack.k-orc.cloud_subnets.yaml

```diff
@@ -106,15 +107,20 @@ spec:
                         minLength: 1
                         type: string
                       gatewayIP:
+                        description: gatewayIP is the IP address of the gateway of
+                          the existing resource
                         maxLength: 45
                         minLength: 1
                         type: string
                       ipVersion:
+                        description: ipVersion of the existing resource
                         enum:
                         - 4
                         - 6
+                        format: int32
```

Comment:
Interesting that `int8` didn't have a format, but `int32` does.

Reply:
`int8` isn't supported by the schema in Kube, all ints must be int32, or perhaps int64 if you really need that size